kubernetes/endpoint.go

```diff
@@ -60,9 +60,6 @@ const (
 
 	// NodeContainerEndpoint the /stats/container endpoint
 	NodeContainerEndpoint Endpoint = "/stats/container"
```

Comment:
We could probably remove the /stats/container endpoint as well while we're here if its not too much work. It was deprecated by k8s in 1.18

Reply:
I'm always happy to delete more code.